CTFd/themes/core/static/css/sticky-footer.css

```diff
@@ -11,7 +11,7 @@ body {
 
 .footer {
     position: absolute;
-    bottom: 0;
+    bottom: 1px;
```

Comment:
Needs a comment!

Reply:
Yeah I agree. @balthasar-m want to add a simple comment saying that this prevents scrollbars from showing on pages that don't use the full page height?

Reply:
@balthasar-m I also think that this change belongs in the admin copy of this file?

Reply:
@ColdHeat Both done.